homework/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="main.css">
+  <title>Javascript3</title>
+</head>
+
+<body>
+  <div>
+    <header class="header">
+      <div class="container">
+        <h1>HYF Repositories</h1>
+        <select id="repositories" class="repo-selector">
+          <option>Please choose a option </option>
+        </select>
+      </div>
+    </header>
+    <div id="main" class="container">
+
+      <div class="left-div whiteframe repo-properties">
+
+        <div><label>Repository:</label> <span id="repo-name"></span></div>
+
+        <div><label>Description:</label> <span id="repo-description"></span></div>
+
+        <div><label>Forks:</label> <span id="repo-forks"></span></div>
+
+        <div><label>Updated:</label> <span id="repo-updated"></span></div>
+
+      </div>
+      <div class="right-div whiteframe">
+      
+        <p class="contributor-header">Contributors</p> <span id="contributorElements"></span>
+         <p class="right_div-username"></p>
+         <p class="right_div-repos"></p>
+         <p class="right_div-avatar"></p>
+      </div>
+    </div>
+  </div>
+  
+  <script src="index1.js"></script>
```

Comment:
For now you should combine all your code into 1 file; that will make it much easier. Maybe Sunday we'll see splitting things up.